### ISSUE TITLE
Command terminated by signal 11

### ISSUE THREAD

**Issue by Arthersun** [endernewton/tf-faster-rcnn#355]
There is a few times meeting this error when the train is in 18th or 19th iter. 
At the first time, I thought it was maybe caused by a big learning rate.
But when I changed it from 0.01 to 0.001, it didn't have any effect.

+ echo Logging output to experiments/logs/res101_voc_2007_trainval__res101.txt.2018-06-13_23-22-28
Logging output to experiments/logs/res101_voc_2007_trainval__res101.txt.2018-06-13_23-22-28
+ set +x
+ '[' '!' -f output/res101/voc_2007_trainval/default/res101_faster_rcnn_iter_1000.ckpt.index ']'
+ [[ ! -z '' ]]
+ CUDA_VISIBLE_DEVICES=0
+ time python ./tools/trainval_net.py --weight data/imagenet_weights/res101.ckpt --imdb voc_2007_trainval --imdbval voc_2007_test --iters 1000 --cfg experiments/cfgs/res101.yml --net res101 --set ANCHOR_SCALES '[8,16,32]' ANCHOR_RATIOS '[0.5,1,2]' TRAIN.STEPSIZE '[500]'
WARNING:tensorflow:From /home/sun/anaconda3/envs/tensorflow/lib/python3.6/site-packages/tensorflow/contrib/learn/python/learn/datasets/base.py:198: retry (from tensorflow.contrib.learn.python.learn.datasets.base) is deprecated and will be removed in a future version.
Instructions for updating:
Use the retry module or similar alternatives.
Called with args:
Namespace(cfg_file='experiments/cfgs/res101.yml', imdb_name='voc_2007_trainval', imdbval_name='voc_2007_test', max_iters=1000, net='res101', set_cfgs=['ANCHOR_SCALES', '[8,16,32]', 'ANCHOR_RATIOS', '[0.5,1,2]', 'TRAIN.STEPSIZE', '[500]'], tag=None, weight='data/imagenet_weights/res101.ckpt')
Using config:
{'TRAIN': {'LEARNING_RATE': 0.001, 'MOMENTUM': 0.9, 'WEIGHT_DECAY': 0.0001, 'GAMMA': 0.1, 'STEPSIZE': [500], 'DISPLAY': 1, 'DOUBLE_BIAS': False, 'TRUNCATED': False, 'BIAS_DECAY': False, 'USE_GT': False, 'ASPECT_GROUPING': False, 'SNAPSHOT_KEPT': 3, 'SUMMARY_INTERVAL': 10, 'SCALES': [600], 'MAX_SIZE': 1000, 'IMS_PER_BATCH': 1, 'BATCH_SIZE': 20, 'FG_FRACTION': 0.25, 'FG_THRESH': 0.5, 'BG_THRESH_HI': 0.5, 'BG_THRESH_LO': 0.0, 'USE_FLIPPED': True, 'BBOX_REG': True, 'BBOX_THRESH': 0.5, 'SNAPSHOT_ITERS': 10, 'SNAPSHOT_PREFIX': 'res101_faster_rcnn', 'BBOX_NORMALIZE_TARGETS': True, 'BBOX_INSIDE_WEIGHTS': [1.0, 1.0, 1.0, 1.0], 'BBOX_NORMALIZE_TARGETS_PRECOMPUTED': True, 'BBOX_NORMALIZE_MEANS': [0.0, 0.0, 0.0, 0.0], 'BBOX_NORMALIZE_STDS': [0.1, 0.1, 0.2, 0.2], 'PROPOSAL_METHOD': 'gt', 'HAS_RPN': True, 'RPN_POSITIVE_OVERLAP': 0.7, 'RPN_NEGATIVE_OVERLAP': 0.3, 'RPN_CLOBBER_POSITIVES': False, 'RPN_FG_FRACTION': 0.5, 'RPN_BATCHSIZE': 100, 'RPN_NMS_THRESH': 0.7, 'RPN_PRE_NMS_TOP_N': 12000, 'RPN_POST_NMS_TOP_N': 2000, 'RPN_BBOX_INSIDE_WEIGHTS': [1.0, 1.0, 1.0, 1.0], 'RPN_POSITIVE_WEIGHT': -1.0, 'USE_ALL_GT': True}, 'TEST': {'SCALES': [600], 'MAX_SIZE': 1000, 'NMS': 0.3, 'SVM': False, 'BBOX_REG': True, 'HAS_RPN': True, 'PROPOSAL_METHOD': 'gt', 'RPN_NMS_THRESH': 0.7, 'RPN_PRE_NMS_TOP_N': 6000, 'RPN_POST_NMS_TOP_N': 300, 'MODE': 'nms', 'RPN_TOP_N': 5000}, 'RESNET': {'MAX_POOL': False, 'FIXED_BLOCKS': 1}, 'MOBILENET': {'REGU_DEPTH': False, 'FIXED_LAYERS': 5, 'WEIGHT_DECAY': 4e-05, 'DEPTH_MULTIPLIER': 1.0}, 'PIXEL_MEANS': array([[[102.9801, 115.9465, 122.7717]]]), 'RNG_SEED': 3, 'ROOT_DIR': '/home/sun/tf-faster-rcnn', 'DATA_DIR': '/home/sun/tf-faster-rcnn/data', 'MATLAB': 'matlab', 'EXP_DIR': 'res101', 'USE_GPU_NMS': True, 'USE_E2E_TF': True, 'POOLING_MODE': 'crop', 'POOLING_SIZE': 7, 'ANCHOR_SCALES': [8, 16, 32], 'ANCHOR_RATIOS': [0.5, 1, 2], 'RPN_CHANNELS': 512}
{'ANCHOR_RATIOS': [0.5, 1, 2],
 'ANCHOR_SCALES': [8, 16, 32],
 'DATA_DIR': '/home/sun/tf-faster-rcnn/data',
 'EXP_DIR': 'res101',
 'MATLAB': 'matlab',
 'MOBILENET': {'DEPTH_MULTIPLIER': 1.0,
               'FIXED_LAYERS': 5,
               'REGU_DEPTH': False,
               'WEIGHT_DECAY': 4e-05},
 'PIXEL_MEANS': array([[[102.9801, 115.9465, 122.7717]]]),
 'POOLING_MODE': 'crop',
 'POOLING_SIZE': 7,
 'RESNET': {'FIXED_BLOCKS': 1, 'MAX_POOL': False},
 'RNG_SEED': 3,
 'ROOT_DIR': '/home/sun/tf-faster-rcnn',
 'RPN_CHANNELS': 512,
 'TEST': {'BBOX_REG': True,
          'HAS_RPN': True,
          'MAX_SIZE': 1000,
          'MODE': 'nms',
          'NMS': 0.3,
          'PROPOSAL_METHOD': 'gt',
          'RPN_NMS_THRESH': 0.7,
          'RPN_POST_NMS_TOP_N': 300,
          'RPN_PRE_NMS_TOP_N': 6000,
          'RPN_TOP_N': 5000,
          'SCALES': [600],
          'SVM': False},
 'TRAIN': {'ASPECT_GROUPING': False,
           'BATCH_SIZE': 20,
           'BBOX_INSIDE_WEIGHTS': [1.0, 1.0, 1.0, 1.0],
           'BBOX_NORMALIZE_MEANS': [0.0, 0.0, 0.0, 0.0],
           'BBOX_NORMALIZE_STDS': [0.1, 0.1, 0.2, 0.2],
           'BBOX_NORMALIZE_TARGETS': True,
           'BBOX_NORMALIZE_TARGETS_PRECOMPUTED': True,
           'BBOX_REG': True,
           'BBOX_THRESH': 0.5,
           'BG_THRESH_HI': 0.5,
           'BG_THRESH_LO': 0.0,
           'BIAS_DECAY': False,
           'DISPLAY': 1,
           'DOUBLE_BIAS': False,
           'FG_FRACTION': 0.25,
           'FG_THRESH': 0.5,
           'GAMMA': 0.1,
           'HAS_RPN': True,
           'IMS_PER_BATCH': 1,
           'LEARNING_RATE': 0.001,
           'MAX_SIZE': 1000,
           'MOMENTUM': 0.9,
           'PROPOSAL_METHOD': 'gt',
           'RPN_BATCHSIZE': 100,
           'RPN_BBOX_INSIDE_WEIGHTS': [1.0, 1.0, 1.0, 1.0],
           'RPN_CLOBBER_POSITIVES': False,
           'RPN_FG_FRACTION': 0.5,
           'RPN_NEGATIVE_OVERLAP': 0.3,
           'RPN_NMS_THRESH': 0.7,
           'RPN_POSITIVE_OVERLAP': 0.7,
           'RPN_POSITIVE_WEIGHT': -1.0,
           'RPN_POST_NMS_TOP_N': 2000,
           'RPN_PRE_NMS_TOP_N': 12000,
           'SCALES': [600],
           'SNAPSHOT_ITERS': 10,
           'SNAPSHOT_KEPT': 3,
           'SNAPSHOT_PREFIX': 'res101_faster_rcnn',
           'STEPSIZE': [500],
           'SUMMARY_INTERVAL': 10,
           'TRUNCATED': False,
           'USE_ALL_GT': True,
           'USE_FLIPPED': True,
           'USE_GT': False,
           'WEIGHT_DECAY': 0.0001},
 'USE_E2E_TF': True,
 'USE_GPU_NMS': True}
Loaded dataset `voc_2007_trainval` for training
Set proposal method: gt
Appending horizontally-flipped training examples...
voc_2007_trainval gt roidb loaded from /home/sun/tf-faster-rcnn/data/cache/voc_2007_trainval_gt_roidb.pkl
done
Preparing training data...
done
1320 roidb entries
Output will be saved to `/home/sun/tf-faster-rcnn/output/res101/voc_2007_trainval/default`
TensorFlow summaries will be saved to `/home/sun/tf-faster-rcnn/tensorboard/res101/voc_2007_trainval/default`
Loaded dataset `voc_2007_test` for training
Set proposal method: gt
Preparing training data...
voc_2007_test gt roidb loaded from /home/sun/tf-faster-rcnn/data/cache/voc_2007_test_gt_roidb.pkl
done
340 validation roidb entries
Filtered 122 roidb entries: 1320 -> 1198
Filtered 33 roidb entries: 340 -> 307
2018-06-13 23:22:43.152362: I tensorflow/core/platform/cpu_feature_guard.cc:140] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
Solving...
/home/sun/anaconda3/envs/tensorflow/lib/python3.6/site-packages/tensorflow/python/ops/gradients_impl.py:100: UserWarning: Converting sparse IndexedSlices to a dense Tensor of unknown shape. This may consume a large amount of memory.
  "Converting sparse IndexedSlices to a dense Tensor of unknown shape. "
Restoring model snapshots from /home/sun/tf-faster-rcnn/output/res101/voc_2007_trainval/default/res101_faster_rcnn_iter_10.ckpt
Restored.
iter: 11 / 1000, total loss: 3.255329
 >>> rpn_loss_cls: 0.589913
 >>> rpn_loss_box: 0.815584
 >>> loss_cls: 0.888034
 >>> loss_box: 0.578349
 >>> lr: 0.010000
speed: 41.041s / iter
iter: 12 / 1000, total loss: 5.695842
 >>> rpn_loss_cls: 1.729684
 >>> rpn_loss_box: 2.931555
 >>> loss_cls: 0.650885
 >>> loss_box: 0.000000
 >>> lr: 0.010000
speed: 41.162s / iter
iter: 13 / 1000, total loss: 8.129940
 >>> rpn_loss_cls: 0.585302
 >>> rpn_loss_box: 0.650503
 >>> loss_cls: 5.625590
 >>> loss_box: 0.884512
 >>> lr: 0.010000
speed: 38.889s / iter
iter: 14 / 1000, total loss: 2.619578
 >>> rpn_loss_cls: 0.418742
 >>> rpn_loss_box: 1.098616
 >>> loss_cls: 0.427641
 >>> loss_box: 0.290123
 >>> lr: 0.010000
speed: 37.372s / iter
iter: 15 / 1000, total loss: 1.245334
 >>> rpn_loss_cls: 0.699922
 >>> rpn_loss_box: 0.030685
 >>> loss_cls: 0.129567
 >>> loss_box: 0.000000
 >>> lr: 0.010000
speed: 36.406s / iter
iter: 16 / 1000, total loss: 10.952494
 >>> rpn_loss_cls: 0.803428
 >>> rpn_loss_box: 5.265312
 >>> loss_cls: 3.002300
 >>> loss_box: 1.495382
 >>> lr: 0.010000
speed: 35.669s / iter
iter: 17 / 1000, total loss: 11.371071
 >>> rpn_loss_cls: 0.751820
 >>> rpn_loss_box: 0.945179
 >>> loss_cls: 8.542075
 >>> loss_box: 0.744810
 >>> lr: 0.010000
speed: 35.033s / iter
iter: 18 / 1000, total loss: 9.369943
 >>> rpn_loss_cls: 0.710904
 >>> rpn_loss_box: 0.965123
 >>> loss_cls: 6.108119
 >>> loss_box: 1.196584
 >>> lr: 0.010000
speed: 34.666s / iter
Command terminated by signal 11
861.84user 200.60system 5:41.27elapsed 311%CPU (0avgtext+0avgdata 1959220maxresident)k
0inputs+0outputs (0major+11895606minor)pagefaults 0swaps

**Comment by rnsandeep** [comment]
Do you have a gpu? it shouldn't usually take 34 sec for 1 iteration. Error is because of memory. Make sure you have sufficient memory atleast 4gb of gpu memory. You can see using nvidia-smi.
